src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -1237,7 +1246,17 @@ private void VisitNode(ParseNode node)
         private void VisitStartStatement(ParseNode node)
         {
             AddFunctionJumpVars(null);
-            VisitChildNodes(node);
```

Comment:
Had to explode this out so I could insert the OpcodeArgBottom at the correct index within the list where the last DECLARE PARAMETER statement had happened. The exploded-out version is in the lines that follow below.